Models/Connect/RemoteCategory.php

```diff
@@ -97,34 +97,36 @@ public function setLabel($label)
     }
 
     /**
-     * @return int
+     * @return bool
      */
-    public function getLocalCategoryId()
+    public function hasLocalCategories()
     {
-        return $this->localCategoryId;
+        return !$this->localCategories->isEmpty();
     }
 
     /**
-     * @param int $id
+     * @return array
      */
-    public function setLocalCategoryId($id)
+    public function getLocalCategories()
     {
-        $this->localCategoryId = $id;
+        return $this->localCategories->toArray();
     }
 
     /**
-     * @return \Shopware\Models\Category\Category
+     * @param \Shopware\Models\Category\Category $localCategory
      */
-    public function getLocalCategory()
+    public function addLocalCategory($localCategory)
```

Comment:
Please use typehint. It must be ```public function addLocalCategory(Category $localCategory)``` where Category is Shopware\Models\Category\Category.